andela/kiba-rc
# #147423545 Implement sorting and filtering

#### What does this PR do?
Here, the user gets to filter by shop and product tag
#### Description of Task to be completed?
Options were setup for the user to be able to refine their search results using filters based on shop and tag
#### How should this be manually tested?
Attempt making a product search and the available filters would appear on the left hand of the search screen
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
147423545
#### Screenshots (if appropriate)
<img width="1091" alt="screen shot 2017-08-03 at 6 07 33 pm" src="https://user-images.githubusercontent.com/28448041/28933871-b119923e-7876-11e7-825a-8c6d4f63e3d1.png">

#### Questions:
n/a
